main/com.spotify/index.js

```diff
@@ -1027,6 +1027,35 @@ module.exports = class SpotifyDevice extends Tp.BaseDevice {
         }
     }
 
+    async do_add_song_to_playlist({
+        song,
+        playlist
+    }) {
+        if (this.state.product !== "premium" && this.state.product !== undefined)
+            throwError("non_premium_account");
```

Comment:
Does one need premium for playlists?

Reply:
I checked and turns out you don't. I'll correct this.